logever/src/main/java/ai/smartfac/logever/controller/FormController.java

```diff
@@ -24,15 +26,33 @@ public ResponseEntity<?> getForms() {
     }
 
     @GetMapping("/{form}/")
-    public ResponseEntity<?> getRole(@PathVariable(name = "form") String form) {
+    public ResponseEntity<?> getForm(@PathVariable(name = "form") String form) {
```

Comment:
Don't we need getFormById as well?

Reply:
We do have form by id. This is when we click on formname from form dashboard to get the form details.